src/sass/components/_timeline.scss

```diff
@@ -0,0 +1,215 @@
+// Copyright (C) 2018 The Trustees of Indiana University
+// SPDX-License-Identifier: BSD-3-Clause
+
+.#{$prefix}-timeline {
+  position: relative;
+  padding: $lg 0;
+
+  &::before {
+    content: '';
+    margin-left: $xxs;
+    position: absolute;
+    top: 0;
+    left: 0;
+    height: 100%;
+    width: $xxs;
+    background: $color-black--200;
+  }
+
+  &__item::after {
+    margin-left: 0;
+    border: solid transparent;
+    content: " ";
+    height: 0;
+    width: 0;
+    position: absolute;
+    pointer-events: none;
+    border-right-color: $color-black--100;
+    border-width: $sm;
+    top: .4rem;
+  }
+
+  &__item {
+    position: relative;
+    margin: $lg 0;
+  }
+
+  /**
+   *  The marker for the timeline visually only looks good at 12px (.75rem). This results in several magic numbers related to the placement of other items such as the &__item::after (above) and date labels (where applicable). This also results in cascading magic numbers for placement of elements such as &__content for variant timelines.
+   */
+
+  &__marker {
+    width: $sm*.75;
+    height: $sm*.75;
+    position: absolute;
+    color: $color-white;
+    background: $color-midnight;
+    border-radius: 50%;
+    box-shadow: 0 0 0 $xxs $color-black--100;
+    margin-top: $sm;
+  }
+
+  &__content {
+    padding: $md;
+    margin-left: $lg;
+    background-color: $color-white;
+    border-radius: $xxs;
+    box-shadow: $shadow-base;
+    border: none;
+  }
+
+  & h2 {
```

Comment:
Generally speaking, if you see generic element selectors inside a component that means the style is too tightly coupled to some specific markup. In this case, the component will only work if the heading happens to be an `h2` element. Depending on the site/application an `h2` might not be correct in the page outline.

Since we're already adding a `rvt-timeline__date` element inside here you could do something like `rvt-timeline__heading` for what is currently an `h2`. That way the developer can use what ever heading level they need to use and the styling can remain consistent.